MQuince/MQuince.Integration.Services.Implementation/HomeService.cs

```diff
@@ -0,0 +1,124 @@
+﻿
+using iTextSharp.text;
+using iTextSharp.text.pdf;
+using iTextSharp.text.pdf.qrcode;
+using MQuince.Integration.Services.Constracts.Interfaces;
+using QRCoder;
+using System;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+
+namespace MQuince.Integration.Services.Implementation
+{
+    public class HomeService 
+    {       
+        public static void GeneratePDF(string name, string jmbg, string medication, string description, MemoryStream ms, QRCoder.QRCode qrCode)
```

Comment:
Every section of pdf could be in separate method, eg. you can create:
private Paragraph GenerateHeaderSection(...data){ ... return header;}
Then in GeneratePDF call, document.Add(GenerateHeaderSection(..data))